### PR TITLE
OCPVE-406: Cluster builds

### PR DESCRIPTION
**Pull request by jeff-roche**

### CODE REVIEW

**Comment by brandisher** [comment]
/lgtm

Keeping the hold for @qJkee's review

**Comment by brandisher** [comment]
Changes are a noop so readding label

/lgtm

File: hack/cluster-deploy.sh
[the remark this answers is not in the document]
Comment (jeff-roche) [reply hack/cluster-deploy.sh:1]: 🤦 deleted the contents but left the file

**Comment by jeff-roche** [comment]
/override ci/prow/lvm-operator-bundle-e2e-aws manual e2e tests are working

**Comment by jeff-roche** [comment]
/override ci/prow/lvm-operator-bundle-e2e-aws

manual e2e tests are working

**Pull request merged**